dist/yoast-seo.js

```diff
@@ -334,7 +337,11 @@ YoastSEO.Analyzer.prototype.linkCount = function() {
 		for ( var i = 0; i < linkMatches.length; i++ ) {
 			var linkKeyword = this.linkKeyword( linkMatches[ i ] );
 			if ( linkKeyword ) {
-				linkCount.totalKeyword++;
+				if ( this.config.keyword !== "" ) {
```

Comment:
Why do we need to do a seperate check for totalKeyword and totalNAKeyword? If we always want to know the total link count, we don't need to check if there's a keyword set or not.